src/modules/Experience/scenes/AboutScene/components/MovingFace/MovingFace.tsx

```diff
@@ -26,33 +24,22 @@ const MovingFace = ({ scenePositionY, selectedColor }: MovingFaceProps) => {
     rotationIntensityY: { value: 0.2, step: 0.001, min: 0.001, max: 2 },
   });
 
-  useFrame((state, delta) => {
-    // Mouse coordinates from -1 to 1
-    const mouse = state.pointer;
-
-    const mouseX = mouse.x;
-    const mouseY = mouse.y;
-
-    // Update mesh position based on mouse position
-    if (faceRef.current) {
-      faceRef.current.position.x = mouseX * positionIntensityX;
-      // ScenePositionY is to correct the position of the face.
-
-      faceRef.current.position.y = scenePositionY + positionY + mouseY * positionIntensityY;
-
-      // Make the face look at the mouse
-      faceRef.current.rotation.x = -mouseY * rotationIntensityX;
-      faceRef.current.rotation.y = mouseX * rotationIntensityY;
-      //faceRef.current.lookAt(mouseX * lookAtIntensity, mouseY * lookAtIntensity, 20);
-    }
-  });
+  const musicNoteMap = new THREE.TextureLoader().load('/textures/movingFace/note.png')
+  /**
+   * Model created by me and
+   * inspired from this tutorial ↓
+   * https://youtu.be/AlPPYkZg9D4?si=L_chsWglPCB2DAGk
+   */
+  const model = useGLTF('/models/head-3.glb')
+  // Let's ignore the issue for now
+  // @ts-ignore
+  const modelGeometry = model.nodes["Head"].geometry.clone()
```

Comment:
To keep things organize, this should be at the Models Folder.

I think we might need a session to organize a bit the files.  Probably me is not understanding XD.  We can do it together.